### PR TITLE
Add hidden text to issue templates

### PR DESCRIPTION
**Pull request by LukasReschke**
Explains that this is a bug tracker + links to owncloud.org/support

@karlitschek @PVince81 @jancborchardt Thoughts?

### CODE REVIEW

**Comment by karlitschek** [comment]
perfect :+1:

File: issue_template.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<!--
```
Comment (oparoz) [inline issue_template.md]: Typo

Comment (nickvergessen) [inline issue_template.md]: hu?

Comment (LukasReschke) [inline issue_template.md]: The + comes from GitHub :wink:

Comment (oparoz) [inline issue_template.md]: Lol... I will be more careful next time :)

**Comment by rullzer** [comment]
:+1:

**Pull request merged**